knative-operator/deploy/resources/console_cli_download_kn_deployment.yaml

```diff
@@ -0,0 +1,141 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+ name: serverless-operator
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: serverless-operator
+  name: kn-cli-downloads
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: serverless-operator
+      component: kn-cli-downloads
+  template:
+    metadata:
+      name: kn-cli-downloads
+      labels:
+        app: serverless-operator
+        component: kn-cli-downloads
+    spec:
+      containers:
+      - name: kn-download-server
+        terminationMessagePolicy: FallbackToLogsOnError
+        image: quay.io/openshift-knative/kn-cli-artifacts:latest
+        imagePullPolicy: Always
```

Comment:
`imagePullPolicy: Always` can go away once we stop using the latest image tag.

Reply:
Removed.